src/style.css

```
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
```

```suggestion
* {
  box-sizing: border-box;
}

html {
    font-family: Roboto, sans-serif;
}
```
move font-family to html selector
do not reset margin and padding for all selectors.
reset just for elements where it is necessary